libs/designsystem/src/lib/components/action-sheet/action-sheet.component.scss

```
@@ -0,0 +1,23 @@
+@import '../../scss/utils';
+
+:host {
+  position: relative;
+  display: inline-block;
+  outline: none;
+}
+
+:host(.is-open) {
+  kirby-action-sheet-popout {
+    display: block;
+  }
+  button {
```

Consider readability:
```suggestion
  }
  
  button {
```

---

fixed